src/tls.c

```diff
@@ -91,12 +96,31 @@ enftun_tls_load_credentials(struct enftun_tls* tls,
     }
     enftun_log_debug("Loaded client TLS certificate %s\n", cert_file);
 
-    if (!(SSL_CTX_use_PrivateKey_file(tls->ctx, key_file, SSL_FILETYPE_PEM) ||
-          SSL_CTX_use_PrivateKey_file(tls->ctx, key_file, SSL_FILETYPE_ASN1)))
+    key_used =
+        SSL_CTX_use_PrivateKey_file(tls->ctx, key_file, SSL_FILETYPE_PEM) ||
+        SSL_CTX_use_PrivateKey_file(tls->ctx, key_file, SSL_FILETYPE_ASN1);
+    if (!key_used)
+        enftun_log_ssl_info("Failed to load client TLS key %s:", key_file);
```

Comment:
Should be an error, not info, message.  Unless the key is a TSS key, in which case we shouldn't log anything here.

My prior suggestion for revising the control flow will implicitly handle the logging correctly too.

Reply:
Factored out with above changes